docs/reference/manifest/versionoverrides-1-0-content.md

```diff
@@ -0,0 +1,43 @@
+---
+title: VersionOverrides 1.0 element in the manifest file for a content add-in
+description: 'Reference documentation of the VersionOverrides element (content) for Office Add-ins manifest (XML) files.'
```

Comment:
I'm curious if our editor @lindalu-MSFT has any suggestions for the description fields of these new articles. The descriptions are very similar, and I find the distinction between them unclear. Perhaps Linda could help clarify.